src/main/java/sirius/db/mixing/properties/MultiPointLocationProperty.java

```diff
@@ -104,7 +107,7 @@ public Object transformFromDatasource(Class<? extends BaseMapper<?, ?, ?>> mappe
             if (coordinates instanceof List<?>) {
                 return ((List<List<Double>>) coordinates).stream()
                                                          .map(entry -> Tuple.create(entry.get(0), entry.get(1)))
-                                                         .toList();
+                                                         .collect(Collectors.toList());
```

Comment:
.collect(Collectors.toCollection(ArrayList::new));
Collectors.toList does not guarantee mutability